logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/handler/ReceiptMessageHandler.kt

```diff
@@ -53,4 +60,16 @@ internal class ReceiptMessageHandlerImpl(
             messageIds = messageContent.messageIds
         )
     }
+
+    private suspend fun updateMessageStatus(
```

Comment:
It uses updateMessage**s**Status in plural form, shouldn't this name also be updateMessage**s**Status?